### DISCUSSION TITLE
Avalara plugin doesn't calculate tax for shipping 0 [Solved, was a misconfiguration]

### DISCUSSION THREAD

**Discussion by DidoMarchet** [saleor/saleor#9606] [General] [answered]
Hi,
we've got an issue with Avalara plugin.

I want to ask if is the correct behaviour for taxes to be not calculated if shipping cost is 0.

Screens below:

Shipping cost == 0 - Taxes == 0
![Screenshot-2022-04-22-at-16 34 41](https://user-images.githubusercontent.com/38940340/164736559-98ed30b3-a683-48bb-ad02-0eecdf4572f7.jpg)

Shipping cost > 0  - Taxes > 0
![Screenshot-2022-04-22-at-16 34 51](https://user-images.githubusercontent.com/38940340/164736755-27e2cdfc-820c-462a-af44-63538751f02d.jpg)

The company says they must add taxes by hand.

Any suggestion are appreciated.

Thanks in advance and kind regards,

Davide

**Comment by korycins** [comment, marked as answer]
Hey @DidoMarchet , 
can you provide an API response for Checkout object?
Is it Saleor3.X? 
When shipping is 0.00, then we don't send the shipping amount to Avatax, but the rest of the objects should have correct taxes.

**Reply by DidoMarchet** [reply]
Hi @korycins, 
sorry for the question but I check and probably the problem was a misconfiguration.

Then I think all works as aspected. 
I check and I close this discussion.
If I have problem I reopen it.

Thanks in advance and kind regards,

Davide
